src/Network/Network/Models/Cortex/PSVpnGateway.cs

```diff
@@ -31,5 +31,8 @@ public class PSVpnGateway : PSTopLevelResource
 
         [Ps1Xml(Label = "Provisioning State", Target = ViewControl.Table)]
         public string ProvisioningState { get; set; }
+
+        [Ps1Xml(Label = "IP Configurations", Target = ViewControl.Table)]
```

Comment:
```suggestion
```
Not a good idea to have too many columns in a table view. Powershell terminal isn't wide usually. Besides this is a complex object.
(Removing `[Ps1Xml]` attribute only removes it from the table view. User can still get the property by `$gateway.IpConfigurations` so it's safe.